packages/main/src/ComboBox.js

```diff
@@ -415,6 +414,13 @@ class ComboBox extends UI5Element {
 		this._itemFocused = false;
 		this.toggleValueStatePopover(this.shouldOpenValueStateMessagePopover);
 		this.storeResponsivePopoverWidth();
+
+		if (this._autocomplete && this.filterValue !== this.value) {
+			this.inner.setSelectionRange(
+				this._isKeyNavigation ? "" : this.filterValue.length,
+				this.value.length
```

Comment:
add a comma in the end

Reply:
I don't get it. Why should a comma be added here?
This is the second argument of the setSelectionRange. The first one is the ternary operation